angular.json

```diff
@@ -148,5 +148,8 @@
       }
     }
   },
-  "defaultProject": "buzz-about"
-}
+  "defaultProject": "buzz-about",
+  "cli": {
+    "analytics": false
```

Comment:
I don't think we should set this for the entire project. It should be an individual choice.

Reply:
Sorry Paul I didn't mean to push that up there was 100 files that were being committed so I did not notice that this was one of them.